cmd/pggen/test/tables_test.go

```diff
@@ -1116,6 +1116,35 @@ func TestNotFoundList(t *testing.T) {
 	}
 }
 
+func TestListSucceedOnPartial(t *testing.T) {
+	txClient, err := pgClient.BeginTx(ctx, nil)
+	chkErr(t, err)
+	defer func() {
+		_ = txClient.Rollback()
+	}()
+
+	entity := models.SmallEntity{
+		Anint: 1892,
+	}
+	smallEntityID, err := txClient.InsertSmallEntity(ctx, &entity)
+	chkErr(t, err)
+
+	params := [][]int64{
+		{23423, smallEntityID}, // partial match
+		{23423},                // just completely missing
+	}
+
+	for _, p := range params {
+		// a partial match will be tagged with NotFoundError
```

Comment:
Big fan of using the array to define a bunch of test cases. The go standard library provides support for easily turning each of these into a subtest with [`testing.T.Run`](https://pkg.go.dev/testing#T.Run). That way you can focus specifically on that test with the `go test -run` flag.